python_modules/dagster-graphql/dagster_graphql/schema/instigation.py

```diff
@@ -177,13 +177,16 @@ class GrapheneDryRunInstigationTick(graphene.ObjectType):
     class Meta:
         name = "DryRunInstigationTick"
 
-    def __init__(self, selector: Union[ScheduleSelector, SensorSelector], timestamp: Optional[float], cursor: Optional[str] = None):
+    def __init__(
+        self,
+        selector: Union[ScheduleSelector, SensorSelector],
+        timestamp: Optional[float],
+        cursor: Optional[str] = None,
+    ):
         self._selector = check.inst_param(selector, "selector", (ScheduleSelector, SensorSelector))
         self._timestamp = timestamp
         self._cursor = cursor
 
-    def resolve_timestamp(self, graphene_info: ResolveInfo):
```

Comment:
You removed this by accident and its causing the type generation to think the timestamp field doesnt exist
<img width="1129" alt="Screen Shot 2023-02-02 at 11 54 21 AM" src="https://user-images.githubusercontent.com/2286579/216390419-e21156c7-451b-42f8-ad27-b417bdc0619f.png">

Reply:
hrm I didn't directly remove it from the gql layer so not sure what's happening here. Will investigate.

Reply:
removing `def resolve_timestamp` is akin to removing it from the gql layer

Reply:
Should be resolved now tho